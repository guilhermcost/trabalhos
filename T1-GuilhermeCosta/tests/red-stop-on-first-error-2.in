is this! an enum
9,ai &p  error?
// the end


