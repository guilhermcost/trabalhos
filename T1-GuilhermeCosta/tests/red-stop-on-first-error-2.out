(1,ID,"is")
(1,ID,"this")
(1,ERROR,"!")
(1,ID,"an")
(1,ID,"enum")
(2,NUM,"9")
(2,SYM,",")
(2,ID,"ai")
(2,ERROR,"&")
(2,ID,"p")
(2,ID,"error")
(2,ERROR,"?")
